scripts/config.ts

```diff
@@ -292,7 +298,7 @@ function writeConfigs(argv: any) {
     if (argv.simple) {
         let simpleConfig = JSON.parse(baseConfJSON)
         simpleConfig.node.staker.enable = true
-        simpleConfig.node.staker["use-smart-contract-wallet"] = true
+        simpleConfig.node.staker["use-smart-contract-wallet"] = false
```

Comment:
TODO: use-smart-contract-wallet is not working because nitro send gas estimate from address(0), so 1) it required address(0) to have eth AND 2) address(0) calling the validator wallet will revert with `NotExecutorOrOwner`